hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/tenantconfiguration/TenantConfigurationDashboardView.java

```diff
@@ -265,93 +129,27 @@ private HorizontalLayout saveConfigurationButtonsLayout() {
-                    configWindowBean.isActionAutoclose());
-        }
-
-        if (configWindowBean.isMultiAssignments()
-                && !readConfigOption(TenantConfigurationKey.MULTI_ASSIGNMENTS_ENABLED)) {
-            writeConfigOption(TenantConfigurationKey.MULTI_ASSIGNMENTS_ENABLED, configWindowBean.isMultiAssignments());
-            repositoryConfigurationView.disableMultipleAssignmentOption();
-        }
-        writeConfigOption(TenantConfigurationKey.AUTHENTICATION_MODE_HEADER_ENABLED,
-                configWindowBean.isCertificateAuth());
-        if (configWindowBean.isCertificateAuth()) {
-            final String value = configWindowBean.getCaRootAuthority() != null ? configWindowBean.getCaRootAuthority()
-                    : "";
-            writeConfigOption(TenantConfigurationKey.AUTHENTICATION_MODE_HEADER_AUTHORITY_NAME, value);
-        }
-
-        if (configWindowBean.isPollingTime()) {
-            tenantConfigurationManagement.addOrUpdateConfiguration(TenantConfigurationKey.POLLING_TIME_INTERVAL,
-                    DurationHelper.durationToFormattedString(configWindowBean.getPollingTimeDuration()));
-        } else {
-            tenantConfigurationManagement.deleteConfiguration(TenantConfigurationKey.POLLING_TIME_INTERVAL);
-        }
-
-        if (configWindowBean.isPollingOverdue()) {
-            tenantConfigurationManagement.addOrUpdateConfiguration(TenantConfigurationKey.POLLING_OVERDUE_TIME_INTERVAL,
-                    DurationHelper.durationToFormattedString(configWindowBean.getPollingOverdueDuration()));
-        } else {
-            tenantConfigurationManagement.deleteConfiguration(TenantConfigurationKey.POLLING_OVERDUE_TIME_INTERVAL);
-        }
-
-        populateAndGetSystemConfig();
-    }
-
-    private boolean readConfigOption(final String configurationKey) {
-        final TenantConfigurationValue<Boolean> enabled = tenantConfigurationManagement
-                .getConfigurationValue(configurationKey, Boolean.class);
-
-        return enabled.getValue() && !enabled.isGlobal();
-    }
-
-    private <T extends Serializable> void writeConfigOption(final String key, final T value) {
-        tenantConfigurationManagement.addOrUpdateConfiguration(key, value);
+        autowiredConfigurationViews.forEach(ConfigurationGroup::save);
     }
 
     private void saveConfiguration() {
-        final boolean isUserInputValid = configurationViews.stream().allMatch(ConfigurationGroup::isUserInputValid);
+        final boolean isUserInputValid = filteredConfigurationViews.stream()
+                .allMatch(ConfigurationGroup::isUserInputValid);
 
         if (!isUserInputValid) {
             uINotification.displayValidationError(i18n.getMessage("notification.configuration.save.notpossible"));
             return;
         }
         saveSystemConfigBean();
-        configurationViews.forEach(ConfigurationGroup::save);
+        filteredConfigurationViews.forEach(ConfigurationGroup::save);
```

Comment:
Check if needed here or can be removed as its already handled in the general save method

Reply:
Done. Removed whole method `saveSystemConfigBean` due to redundancy.